File: Upcoder/Lập trình cơ bản/3.Vòng lặp/Tìm UCLN, BCNN/BT_Cơ_bản2.py
a, b = map(int, input().split())

# Tìm Ước số chung lớn nhất (GCD)
for i in range(a, 0, -1):
    if a % i == 0 and b % i == 0:
        print(i, end=" ")
        break

# Tìm Bội số chung nhỏ nhất (LCM)
for k in range(a, a * b + 1):
    if k % a == 0 and k % b == 0:
        print(k)
        break

File: Upcoder/Lập trình cơ bản/3.Vòng lặp/Tìm UCLN, BCNN/BT_Cơ_bản2.cs
using System;
namespace TimUCLNBCNN
{
 public class Program
    {
        static void Main(string[]args)
        {
            string [] input = Console.ReadLine().Split(' ');
            int a = int.Parse(input[0]);
            int b = int.Parse(input[1]);

            // Tìm Ước số chung lớn nhất (GCD)
            for (int i = a; i > 0; i--)
            {
                if (a % i == 0 && b % i == 0)
                {
                    Console.Write(i + " ");
                    break;
                }
            }

            // Tìm Bội số chung nhỏ nhất (LCM)
            for (int k = a; k <= a * b; k++)
            {
                if (k % a == 0 && k % b == 0)
                {
                    Console.Write(k);
                    break;
                }
            }
        }
    }
}
   
